src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -589,6 +581,14 @@ LRESULT FancyZones::WndProc(HWND window, UINT message, WPARAM wparam, LPARAM lpa
                 m_terminateEditorEvent.release();
             }
         }
+        else if (message == WM_PRIV_VDUPDATE)

```

Comment:
perhaps we should start using `case` labels instead of nested `ifs` inside the `default` label for custom events

Reply:
I agree.

Reply:
Why not have all of them in the same switch. We have named them `WM_PRIV_*` so it's quite clear what are windows defined and what are our internal.

Reply:
yes, that want I meant

Reply:
Sometimes the sentences can be read in two different ways, so good you asked so now it's clear.

Reply:
Hm, now that I look at it, the problem is that our internally defined messages as such cannot be used as labels in switch-case statement. They are not _Integral Constant Expressions_, which is probably reason that this is implemented this way in the first place.

Reply:
that's unfortunate C++ limitation!